app/response_processor.py

```diff
@@ -146,64 +154,84 @@ def send_notification(self, survey_id):
                 self.logger.info("Ignoring received CTP submission")
             else:
                 self.logger.info("About to publish notification to queue")
-                self.notifications.publish_message(self.tx_id, headers={'tx_id': self.tx_id})
+                self.notifications.publish_message(
+                    self.tx_id, headers={
+                        'tx_id': self.tx_id
+                    })
         except PublishMessageError as e:
             self.logger.error("Unable to queue response notification", error=e)
             raise RetryableError
 
     def decrypt_survey(self, encrypted_survey):
         self.logger.info("Decrypting survey")
-        response = self.remote_call(settings.SDX_DECRYPT_URL,
-                                    data=encrypted_survey)
+        response = self.remote_call(
+            settings.SDX_DECRYPT_URL, data=encrypted_survey)
         try:
             self.response_ok(response)
         except ClientError:
-            self.logger.error("Survey decryption unsuccessful. Quarantining Survey.")
+            self.logger.error(
+                "Survey decryption unsuccessful. Quarantining Survey.")
             raise QuarantinableError
 
         self.logger.info("Survey decryption successful")
         return response.json()
 
     def validate_survey(self, decrypted_json):
         self.logger.info("Validating survey")
-        self.response_ok(self.remote_call(settings.SDX_VALIDATE_URL,
-                                          json=decrypted_json))
+        self.response_ok(
+            self.remote_call(settings.SDX_VALIDATE_URL, json=decrypted_json))
         self.logger.info("Survey validation successful")
 
     def store_survey(self, decrypted_json):
         self.logger.info("Storing survey")
-        response = self.remote_call(settings.SDX_RESPONSES_URL,
-                                    json=decrypted_json)
+        response = self.remote_call(
+            settings.SDX_RESPONSES_URL, json=decrypted_json)
         try:
             self.response_ok(response)
         except ClientError:
-            self.logger.error("Survey storage unsuccessful. Quarantining Survey.")
+            self.logger.error(
+                "Survey storage unsuccessful. Quarantining Survey.")
             raise QuarantinableError
 
         self.logger.info("Survey storage successful")
 
-    def remote_call(self, request_url, json=None, data=None, headers=None,
-                    verify=True, auth=None):
+    def remote_call(self,
+                    request_url,
+                    json=None,
+                    data=None,
+                    headers=None,
+                    verify=True,
+                    auth=None):
```

Comment:
I wonder if would not be a btter practice to redue the number of arguments, and sue soemthign like *values or collect them in a single object that will also provide the validation . I am not a big fan of IF condition eather.

Reply:
No point in passing these anyway as remote_call defaults all of these to the same values.  Would remote_call(self, request_url) work in the same way? (I know you didn't change the code in this PR sorry!, feel free to ignore my suggestion ;) )

Reply:
This is outside the scope of this PR - only came up green due to auto-formatting of code.